apps/meeting/webpack.config.js

```diff
@@ -2,7 +2,6 @@
 // SPDX-License-Identifier: MIT-0
 
 const path = require('path');
-const InlineChunkHtmlPlugin = require('react-dev-utils/InlineChunkHtmlPlugin');
```

Comment:
I think we cannot remove this plugin cause this is the one helping us putting all in one HTML file which we then deploy. Can you once check if this still helps us deploy using existing setup?